### ISSUE TITLE
It shouldn't be possible to create already used shortcuts

### ISSUE THREAD

**Issue by TimotheeChauvin** [luoliyan/incremental-reading#51]
(v4.7.14, Anki v2.1beta27, Linux)
For example, I can create a shortcut Alt + H. But when I then try to use this shortcut, the help menu opens.
Already used shortcuts seem to have precedence over newly created ones.

This is admittedly a minor issue, that only arises a few times when first discovering the add-on. But I think that to avoid losing new users, everything must work perfectly from the beginning.

**Comment by luoliyan** [comment]
I agree this should be prevented. I've added it to my to-do list, but as you might guess, there are many items higher in the list at the moment.
